pyteal/ast/widemath.py

```diff
@@ -157,3 +214,45 @@ def has_return(self):
 
 
 WideRatio.__module__ = "pyteal"
+
+
+class WideUint128(MultiValue):
+    @staticmethod
+    def addw(*terms: Expr):
+        if len(terms) < 2:
+            pass
+        return WideUint128(
+            Op.addw, [TealType.uint64, TealType.uint64], args=list(terms)
+        )
+
+    @staticmethod
+    def mulw(*factors: Expr):
+        if len(factors) < 2:
+            pass
+        return WideUint128(
+            Op.addw, [TealType.uint64, TealType.uint64], args=list(factors)
+        )
+
+    @staticmethod
+    def expw(base: Expr, _pow: Expr):
+        return WideUint128(
+            Op.expw, [TealType.uint64, TealType.uint64], args=[base, _pow]
+        )
+
+    def __init__(self, op: Op, types: List[TealType], *, args: List[Expr]):
+        super().__init__(op, types, args=args)
+
+    def __add__(self, other: Expr):
```

Comment:
@ahangsu Clarification question:  How did you decide which intrinsic operators from https://docs.python.org/3/library/operator.html to override here?

I'm trying to gauge if there's a rationale for the subset overridden.  Or if possibly the implementation is partial because the PR is WIP.